airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -89,8 +108,10 @@ class GoogleAdsStream(Stream, ABC):
     CATCH_API_ERRORS = True
 
     def __init__(self, api: GoogleAds, customers: List[Customer]):
+        super().__init__()
```

Comment:
why do we need to invoke the paren't `init` method when we were not before this PR?

Reply:
I overlooked it, thanks. Removed